bin/generate-billings.js

```diff
@@ -56,30 +56,56 @@ const InProgressBilling = {
 const SuccessfulBilling = {
     dv: DV,
     sender: SENDER,
-    name: 'Success',
-    shortDescription: 'Шаблон СБ Бизнес Онлайн 8_2',
-    detailsTitle: 'Подключение реестрового обмена',
-    detailsText: 'Выбрав данный вариант интеграции, вам будет необходимо загрузить ваши реестры к нам самостоятельно',
+    name: 'Интеграция через загрузку вашего реестра (DONE STATUS)',
+    shortDescription: 'Поддерживаемые форматы: 1С, СБ Бизнес Онлайн 8_2 и 9_1',
+    detailsTitle: 'Интеграция через загрузку реестров',
+    detailsText: 'Выберите формат, в котором хотите загружать ваши реестры в Домá.\n' +
+        'Мы запомним ваш выбор и в следующий раз не будем спрашивать про формат файлов.',
     detailsConfirmButtonText: 'Подключить',
-    detailsInstructionButtonText: 'Подробнее про шаблон',
-    detailsInstructionButtonLink: 'https://www.sberbank.ru/ru/s_m_business/new_sbbol',
     contextDefaultStatus: BILLING_INTEGRATION_ORGANIZATION_CONTEXT_FINISHED_STATUS,
     billingPageTitle: 'Биллинг, реестровый обмен',
-    dataFormat: Lvl1DataFormat,
+    dataFormat: Lvl3DataFormat,
+    availableOptions: {
+        title: 'Формат ваших реестров',
+        options: [
+            {
+                name: '1C',
+                displayName: '1C',
+                billingPageTitle: 'Интеграция через загрузку реестров 1С',
+                descriptionDetails: {
+                    detailsText: 'о формате',
+                    detailsLink: 'https://help.doma.ai',
+                },
```

Comment:
Why not use some common JSON format for this? Like ANT?

Reply:
What do you mean by that?

Reply:
We'll describe format in confluence page